packages/integrations/useAxios/index.ts

```diff
@@ -42,12 +42,12 @@ export function useAxios<T = any>(url: string, ...args: any[]) {
       instance = args[1]
   }
 
-  const response = ref<any>(null) as Ref<AxiosResponse<T> | undefined>
-  const data = ref<any>(undefined) as Ref<T | undefined>
+  const response = shallowRef<any>(null) as Ref<AxiosResponse<T> | null>
+  const data = shallowRef<any>(null) as Ref<T | null>
```

Comment:
Since shalowRef does not unwrap, can we just have the type in the generic?

Reply:
Hmm I'm not quite sure what you mean. I did what I thought you were talking about above but maybe that was a misunderstanding? Could we just do something like `const response = shallowRef<AxiosResponse<T>>()`